dp

```diff
@@ -34,11 +36,15 @@ EOF
 
 # Default settings
 PORT=22
-BASE_DIR="\$HOME/app"
+BASE_DIR="\$HOME/dp-apps"
+APP_NAME="app"
```

Comment:
Why add this variable and change the above default? IMO the current setup where you basically just set the full dir name in the app's .config is perfectly sufficient anyway